### PR TITLE
[GraphBolt][Doc] Update `layer_neighbor` documentation for new args.

### PR DESCRIPTION
**Pull request by mfbalin**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] I've leverage the [tools](https://docs.google.com/document/d/1iHyj7zlmygKSk5gBPsqIqL5ASPzJSPREaNT_QdsiYA4/edit) to beautify the python and c++ code.
- [ ] The PR is complete and small, read the [Google eng practice (CL equals to PR)](https://google.github.io/eng-practices/review/developer/small-cls.html) to understand more about small PR. In DGL, we consider PRs with less than 200 lines of core code change are small (example, test and documentation could be exempted).
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

File: python/dgl/graphbolt/impl/fused_csc_sampling_graph.py [hunk 1]
```diff
@@ -791,6 +791,32 @@ def sample_layer_neighbors(
             corresponding to each neighboring edge of a node. It must be a 1D
             floating-point or boolean tensor, with the number of elements
             equalling the total number of edges.
+        random_seed: torch.Tensor, optional
```
Comment (frozenbugs) [inline python/dgl/graphbolt/impl/fused_csc_sampling_graph.py:794]: Add an augment check to raise augment error before calling deeper to C++?

Comment (mfbalin) [reply python/dgl/graphbolt/impl/fused_csc_sampling_graph.py:794]: What kind of a check, like checking whether the random_seed tensor is empty?

Comment (frozenbugs) [reply python/dgl/graphbolt/impl/fused_csc_sampling_graph.py:794]: To check whether the input is `An int64 tensor with one or two elements.` or not.

Comment (frozenbugs) [reply python/dgl/graphbolt/impl/fused_csc_sampling_graph.py:794]: same to seed2_contribution `A float value between [0, 1)`

**Review by frozenbugs — APPROVED** [review]

File: python/dgl/graphbolt/impl/fused_csc_sampling_graph.py (hunk 1 above)
Comment (mfbalin) [reply python/dgl/graphbolt/impl/fused_csc_sampling_graph.py:794]: Added. Could you check and merge if the checks look good?

**Review by frozenbugs — APPROVED** [review]

**Pull request merged**